src/styles/main.scss

```
@@ -1,3 +1,77 @@
+$cell-size: 100px;
+$calendar-gap: 1px;
+$calendar-padding: 10px;
+$calendar-max-width: 2 * $calendar-padding + 7 * $cell-size + 6 * $calendar-gap;
+
+$start-day:
+  "mon" 0,
+  "tue" ($cell-size + $calendar-gap),
+  "wed" (2 * $cell-size + 2 * $calendar-gap),
```

You can only calculate your margin once, instead of calculating it every time
```suggestion
  "tue" $day-margin,
  "wed" 2 * $day-margin,
```